static-assets/components/cstudio-admin/mods/content-types.js

```diff
@@ -3742,7 +3742,11 @@
 
     CStudioAdminConsole.helpInsert = function (button, identifier) {
       var $button = $(button);
-      var $input = $(identifier).siblings('input');
+      // Some prop-types wrap the input within a div
+      const $input =
+        $(identifier).siblings('div').length > 0
+          ? $(identifier).siblings('div').find('input')
+          : $(identifier).siblings('input');
```

Comment:
Too wired. Go to the parent of the row and find the child there. Preferably through some specific class or attribute if available.

Reply:
Right now theres no class to find directly the input (there are different types of controls with different classes). By using `identifier` (is a selector), getting the parent and finding the input it does work. Should we add a common class to the controls (input) for a more specific selection?

Reply:
Tested with current usages (postfixes and macros) and it is working